demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -920,6 +934,7 @@ CK_RV xProvisionDevice( CK_SESSION_HANDLE xSession,
     ProvisionedState_t xProvisionedState = { 0 };
     CK_OBJECT_HANDLE xObject = 0;
     CK_BBOOL xImportedPrivateKey = CK_FALSE;
+    CK_BBOOL xGeneratedPrivateKey = CK_FALSE;
```

Comment:
Can we explicitly initialized these:

```
xProvisionedState.xPrivateKey = CK_INVALID_HANDLE;
xProvisionedState.xClientCertificate = CK_INVALID_HANDLE;
xProvisionedState.xPublicKey = CK_INVALID_HANDLE;
```

This is only for readability as ```CK_INVALID_HANDLE``` is defined to ```0```.